openstudiocore/src/model/AirTerminalSingleDuctConstantVolumeReheat.cpp

```diff
@@ -661,6 +687,16 @@ void AirTerminalSingleDuctConstantVolumeReheat::resetMaximumReheatAirTemperature
   getImpl<detail::AirTerminalSingleDuctConstantVolumeReheat_Impl>()->resetMaximumReheatAirTemperature();
 }
 
+boost::optional<AirflowNetworkEquivalentDuct> AirTerminalSingleDuctConstantVolumeReheat::createAirflowNetworkEquivalentDuct(double length, double diameter)
```

Comment:
I see here why you want to name this "create" because it takes arguments and if you returned the existing you would either have to change those or fail.  Can we have default values for these fields and the user just has to set them on the created duct?

Reply:
In some fancy future world we could estimate duct length by distance between zones or something extra science-y like that